en-US/focus-ios.xliff

```diff
@@ -490,40 +545,6 @@
         <source>$(PRODUCT_NAME)</source>
         <target>$(PRODUCT_NAME)</target>
       </trans-unit>
-      <trans-unit id="CFBundleShortVersionString">
```

Comment:
Same as the other removed string. I have no clue if this is OK, that's an iOS development question.